i18n/en.json

```diff
@@ -2256,27 +2256,27 @@
   },
   {
     "id": "api.user.verify_email.broken_token.app_error",
-    "translation": ""
+    "translation": "Bad verify email token type."
   },
   {
     "id": "api.web_socket.connect.upgrade.app_error",
     "translation": "Failed to upgrade websocket connection"
   },
   {
     "id": "api.web_socket_router.bad_action.app_error",
-    "translation": ""
+    "translation": "Unknown websocket action."
   },
   {
     "id": "api.web_socket_router.bad_seq.app_error",
-    "translation": ""
+    "translation": "Invalid websocket message sequence."
   },
   {
     "id": "api.web_socket_router.no_action.app_error",
-    "translation": ""
+    "translation": "No websocket action."
   },
   {
     "id": "api.web_socket_router.not_authenticated.app_error",
-    "translation": ""
+    "translation": "Not authenticated websocket connection."
```

Comment:
`Failed to authenticate WebSocket connection.` (?)

Reply:
It is not exactly that, it is the fact that the connection is not authenticated at all, is not an error trying to authenticate, is an error because the user who tries to do the action is not authenticated yet.